apps/functions/src/app/player.ts

```diff
@@ -15,7 +15,28 @@ const JWPlayerApi = require('jwplatform');
 export const linkDuration = 60 * 60 * 5; // 5 hours in seconds = 60 seconds * 60 minutes * 5 = 18 000 seconds
 
 
-export const checkSignature = async (data: SignedMessage, context: https.CallableContext): Promise<string> => {
+
+export const checkSignature = async (data: SignedMessage, context: https.CallableContext): Promise<PlayerResponse> => {
+
+  const jwplayerKey = config().jwplayer?.key;
```

Comment:
always use HttpError in a callable function to avoid weird error message.
```typescript
const config = https.config().jwplayer;
if (!config) throw new https.HttpError('failed-precondition', 'jwp config is missing');
const { secret, key } =  config;
if (!secret) throw new https.HttpError('failed-precondition', 'jwp secret is missing');
```